### PR TITLE
fix: Uses the latest configuration setup

### LINKED ISSUE #28: Dynaconf deprecation warning
Set static var in settings module like the warning message advises

```bash
venv/lib/python3.9/site-packages/dynaconf/base.py:151
  /home/aravinth/code/forgedfed/interface/venv/lib/python3.9/site-packages/dynaconf/base.py:151: DeprecationWarning: Usage of `from dynaconf import settings` is now DEPRECATED in 3.0.0+. You are encouraged to change it to your own instance e.g: `settings = Dynaconf(*options)`
    warnings.warn(
```

### PR DESCRIPTION
**Pull request by dat-adi**
This PR fixes #28.
## Changelog
Dynaconf threw a warning about deprecated import methodology for `interface`, so updated it.

- [x] Dynaconf configuration upgrade.
- [x] Simple black format.
- [x] Ignore `.secrets.toml`.
- [x] Set up an environment prefix for interface.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,12 +17,12 @@
 from setuptools import find_packages, setup
 
 setup(
-    name='interface',
-    version='0.1.0',
+    name="interface",
```
Comment (realaravinth) [inline setup.py:20]: is there a reason why double quotes are used here?

File: .gitignore [hunk 2]
```diff
@@ -154,3 +154,4 @@ grcov.tar.bz2
 lcov.info
 *.profraw
 lcov.info
+config/.secrets.toml
```
Comment (realaravinth) [inline .gitignore:157]: [`.env``](https://github.com/forgeflux-org/interface/blob/146a2cd7f5afc93fdd9ff3affc26b8770a412f1b/config/.env-sample) file serve the same purpose of `.secrets.toml`. It's the alternative to passing secrets via env vars

File: interface/__main__.py [hunk 3]
```diff
@@ -19,7 +19,11 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import time
 from threading import Thread
-from dynaconf import settings
+from dynaconf import Dynaconf
+
+settings = Dynaconf(
```
Comment (realaravinth) [inline interface/__main__.py:24]: As mentioned in matrix, loading `Dynaconf` in every module can add filesystem overheads.

Loading it once and re-exporting it should solve this issue.

`itnerface/settings.py`:

```python
from dynaconf import Dynaconf

settings = Dynaconf(
    environments=True, settings_files=["settings.toml"]
)
```

to use `interface.settings.py`, the following change is recommended:
`interface/ns.py` diff
```diff
diff --git a/interface/ns.py b/interface/ns.py
index ea8ab1f..d3afec1 100644
--- a/interface/ns.py
+++ b/interface/ns.py
@@ -21,7 +21,7 @@ from functools import lru_cache

 import requests

-from dynaconf import settings
+from interface.settings import settings
 from interface.utils import clean_url, since_epoch, trim_url
 from interface.error import Error
```

**Review by realaravinth — CHANGES_REQUESTED** [review]
Thank you for working on #28, it has been around forever.

File: setup.py (hunk 1 above)
Comment (dat-adi) [reply setup.py:20]: No specific reason, this is the result of linting using the `black` formatter.

Comment (realaravinth) [reply setup.py:20]: I seee. I just noticed that `setup.py` isn't included in the `lint` subroutine in Makefile, can you please make that change?

**Comment by dat-adi** [comment]
Made changes as requested, the variables are now loaded from the instance settings, rather than being called from file for every file.

**Review by realaravinth — CHANGES_REQUESTED** [review]
Please see https://github.com/forgeflux-org/interface/pull/89#discussion_r802393337

File: setup.py (hunk 1 above)
Comment (dat-adi) [reply setup.py:20]: Yup, the change has been made.

**Review by realaravinth — APPROVED** [review]

**Pull request merged**